SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceUniqueId.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_trace.h>
+
+class UFunction;
+
+namespace SpatialGDK
+{
+struct EventTraceUniqueId
+{
+	uint32 Hash{ 0 };
+	EventTraceUniqueId(uint32 Hash)
+		: Hash(Hash)
+	{
+	}
+
+	FString ToString() const;
+
+	bool IsValid() const { return Hash != 0; }
+
+	static EventTraceUniqueId Generate(uint64 Entity, uint8 Type, uint64 Id);
```

Comment:
Nit: Maybe we should be explicit and say this EventTraceUniqueId is for RPCs and have Generate take the rpc type and rpc id? It doesn't seem like it is very generic as is?